subsys/bluetooth/audio/media_proxy.c

```diff
@@ -59,13 +59,13 @@ struct mprx {
 #if defined(CONFIG_MCTL_REMOTE_PLAYER_CONTROL)
 	/* Remote media player - to have a player instance for the user */
 	struct media_player  remote_player;
+
+	/* MCC callbacks */
+	struct bt_mcc_cb mcc_cbs;
```

Comment:
These callbacks will still rely on `CONFIG_BT_MCC`, so the guard should still be there.

Reply:
It is - MCTL_REMOTE_PLAYER_CONTROL depends upon BT_MCC.  
(This is how we have done it elsewhere - only use the "strictest"/"innermost" guard.)

And when MCC is rewritten to register as a client, the callbacks will anyway come with the registration.